app/controllers/items_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class ItemsController < ApplicationController
+
+  before_action :set_seller
+
+  def index
+    query = Validate::ItemQuery.new(params)
```

Comment:
This works, but ActionController::Parameters has a .permit and .require method that you can use to validate

Reply:
Yup was planning to use it but I wanted a wrapper to enable me to use `ActiveModel::Validations`